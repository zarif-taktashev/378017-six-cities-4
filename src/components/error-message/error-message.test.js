import * as React from "react";
import * as renderer from "react-test-renderer";
import ErrorMessage from "./error-message";

it(`Render with ErrorMessage`, () => {
  const tree = renderer
    .create(
        <ErrorMessage
          loginError={`Faild to complete action`}
        />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

