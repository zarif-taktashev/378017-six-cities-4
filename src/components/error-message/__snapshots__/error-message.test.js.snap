// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render with ErrorMessage 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "color": "white",
      "left": 0,
      "margin": "auto",
      "padding": "10px 20px",
      "position": "fixed",
      "right": 0,
      "textAlign": "center",
      "top": 0,
      "width": "200px",
    }
  }
>
  Faild to complete action
</div>
`;
